README.md

```diff
@@ -83,17 +83,17 @@ yarn install
 yarn global add gulp
 ```
 
-If you meet permission problem when trying to install yarn globally, check [this](https://github.com/yarnpkg/yarn/issues/1060#issuecomment-268160528) out.
+If you find permission problem when trying to install yarn globally, check [this](https://github.com/yarnpkg/yarn/issues/1060#issuecomment-268160528) out.
 
 ### Setup database and session store
 
-You can either run `postgresql` and `redis` using docker images:
+You can either run `postgresql` or `redis` using docker images:
```

Comment:
should still be and, below has an 'or' (or maintain it by yourself) for the 'either' above

Reply:
Correct it, and added two bullets to make it looks more clear.